maintainers.md

```diff
@@ -0,0 +1,24 @@
+# Maintainers
+
+## Current Maintainers As of 15th of September 2025
+
+- [Mitch Gaffigan](https://github.com/mgaffigan)
```

Comment:
I'm a "yes" on Mitch. I think he's done a superb job not only in the volume and quality of his contributions, but also in assisting with PRs contributed by others, responding to open issues, and offering general assistance to other users and developers in discord.